app/lib/common/widget/AppCommon.dart

```diff
@@ -25,18 +26,14 @@ String getNameFromId(String name) {
   return name.substring(start, end);
 }
 
-String formatedTime(int value) {
-  int h, m;
+String? formatedTime(int value) {
```

Comment:
okay. help me out here. how is this not just a call to [`intl`](https://pub.dev/documentation/intl/latest/intl/DateFormat-class.html)s `DateFormat.hm().format(DateTime.fromMillisecondsSinceEpoch(value, isUtc:true))` ? 

I know I asked for using `time` but simply because I wasn't aware that `DateTime` is in core and formatting can easily be done with `intl`. Let's go this route instead please.

[1]: https://api.dart.dev/stable/2.16.2/dart-core/DateTime/DateTime.fromMillisecondsSinceEpoch.html